### ISSUE TITLE
How to use sample_onnx_mnist.cpp for Fashion mnist dataset

### ISSUE THREAD

**Issue by Knightfire1998** [NVIDIA/TensorRT#641]
## Description
i want to  use onnx mode trained on converted from a pytorch model trained on fashion mnist now how to build the tesnorrt engine using sample_onxx_mnist.cpp

## Environment

**TensorRT Version**: 7.1.3.4
**GPU Type**: Nvidia GeForce 940mx
**Nvidia Driver Version**: 440 
**CUDA Version**: 10.2
**CUDNN Version**: 8.0
**Operating System + Version**: ubuntu 20.04 
**Python Version (if applicable)**: 3.8
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**:

**Comment by rmccorm4** [comment]
Hi @Knightfire1998 ,

Since a FashionMNIST model will likely have the same input and output shapes as a standard MNIST model, you should more or less be able to [swap in the path to your custom ONNX model](https://github.com/NVIDIA/TensorRT/blob/2b8863ddff04fabfe13fa5c201ab04a7e82bd470/samples/opensource/sampleOnnxMNIST/sampleOnnxMNIST.cpp#L167), and [swap in the path to some sample FashionMNIST images](https://github.com/NVIDIA/TensorRT/blob/2b8863ddff04fabfe13fa5c201ab04a7e82bd470/samples/opensource/sampleOnnxMNIST/sampleOnnxMNIST.cpp#L247).

I think you could leave everything else the same and get some simple results.

**Comment by Knightfire1998** [comment]
@rmccorm4  hello i tried that but onxx model had problem with the node name so i changed them but now i have a problem regarding the output accuracy is very less as well as its shows failed after execution i ll attach the ss
![Screenshot from 2020-06-27 00-54-15](https://user-images.githubusercontent.com/41750768/85893646-e0e03900-b810-11ea-9ba0-8e7fcde0f6c1.png)

**Comment by Knightfire1998** [comment]
i tried to make the onnx file n tested it it worked fine but it shows failed in the end and for fashion its accuracy is really poor help me!!!!

**Comment by rmccorm4** [comment]
Hi,

There is probably some hard-coded verification to compare the outputs to the MNIST digits/PGM files. I recommend digging through the code to make it more specific to your Fashion MNIST data.

**Comment by mk-nvidia** [comment]
Closing due to no response from the user.
